x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout.tsx

```diff
@@ -92,38 +108,68 @@ const Cards = ({ data }: { data: Card[] }) => (
 
 const FindingsTab = ({ tab, findings }: { findings: CspFinding; tab: FindingsTab }) => {
   switch (tab) {
-    case 'result':
-      return <Cards data={getResultCards(findings)} />;
-    case 'rule':
-      return <Cards data={getRuleCards(findings)} />;
+    case 'remediation':
+      return <Cards data={getRemediationCards(findings)} />;
     case 'resource':
       return <Cards data={getResourceCards(findings)} />;
+    case 'general':
+      return <Cards data={getGeneralCards(findings)} />;
     default:
       assertNever(tab);
   }
 };
 
-const getResourceCards = ({ resource }: CspFinding): Card[] => [
+const getResourceCards = ({ resource, host }: CspFinding): Card[] => [
   {
     title: TEXT.RESOURCE,
     listItems: [
-      [TEXT.FILENAME, <EuiCode>{resource.filename}</EuiCode>],
+      [TEXT.FILENAME, <CodeBlock>{resource.filename}</CodeBlock>],
       [TEXT.MODE, resource.mode],
-      [TEXT.PATH, <EuiCode>{resource.path}</EuiCode>],
+      [TEXT.PATH, <CodeBlock>{resource.path}</CodeBlock>],
       [TEXT.TYPE, resource.type],
       [TEXT.UID, resource.uid],
     ],
   },
+  {
+    title: TEXT.HOST,
+    listItems: [
+      [TEXT.ARCHITECTURE, host.architecture],
+      [TEXT.CONTAINERIZED, host.containerized ? 'true' : 'false'],
+      [TEXT.HOSTNAME, host.hostname],
+      [TEXT.ID, <CodeBlock>{host.id}</CodeBlock>],
+      [TEXT.IP, <CodeBlock>{host.ip.join(',')}</CodeBlock>],
+      [TEXT.MAC, <CodeBlock>{host.mac.join(',')}</CodeBlock>],
```

Comment:
i think maybe `.join(', ')`
```suggestion
      [TEXT.IP, <CodeBlock>{host.ip.join(', ')}</CodeBlock>],
      [TEXT.MAC, <CodeBlock>{host.mac.join(', ')}</CodeBlock>],
```